src/js/dbSNP-import/dbSNP-import-progress.js

```diff
@@ -102,18 +103,19 @@ EvadbSNPImportProgress.prototype = {
                 taxonomy_link = '<a target="_blank" href="https://www.ebi.ac.uk/ena/data/view/Taxon:'+this[key].taxId+'">'+this[key].taxId+'</a>';
             }
 
-            var importedIds = _this._getImportStatus(this[key].importedIds, this[key].totalIdsDbsnp);
-
-            var importedSynonymousIds = _this._getImportStatus(this[key].importedSynonymousIds, this[key].totalSynonymousIdsDbsnp);            
+            var importedRs = _this._getImportStatus(this[key].importedRs, this[key].totalRsDbsnp);
```

Comment:
The method `getImportStatus` doesn't show the correct total if zero IDs have been imported.